var/spack/repos/builtin/packages/asagi/package.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+#
+# Author: Gilbert Brietzke
+# Date: July 2, 2019
+
+from spack import *
+
+
+class Asagi(CMakePackage):
+    """a pArallel Server for Adaptive GeoInformation."""
+
+    homepage = "https://github.com/TUM-I5/ASAGI"
+    git = "https://github.com/TUM-I5/ASAGI.git"
+
+    version('1.0.1', commit='f633f96931ae00805f599078d5a1a6a830881554',
+            submodules=True, preferred=True)
+    version('1.0', commit='f67250798b435c308b9a1e7516f916f7855534ec',
+            submodules=True)
+
+    variant('link_type', default='shared',
+            description='build shared and/or static libraries',
+            values=('static', 'shared'), multi=True)
+
+    variant('fortran', default=True, description="enable fortran support")
+    variant('maxDimensions', default=4,
```

Comment:
I believe this should be called `max-dimensions`.  Similarly, `link_type` above should be called `link-type` (sorry).  Again, same issue with variants like `threadsafeCounter` and other variant names below. @adamjstewart can you weigh in on this?  (I remember we discussed it and came to decisions, but I can't find the final guidelines right now).

Reply:
do you mean camelCase isnt allowed for variants ? - I think I have seen other specs with camelCase

Reply:
```
$ grep variant */package.py | less
```
Nope, no camel case and no `-`.  If there is, it slipped through.

Reply:
changed naming scheme of variants